### PR TITLE
Splunk double quates bug

### PR DESCRIPTION
**Pull request by eyalpalo**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-39874), [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-42036)

## Description
Fixed an issue where double quotes were unnecessarily removed.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/SplunkPy/ReleaseNotes/3_1_39.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### SplunkPy
+
+- Fixed an issue where double quotes were unnecessarily removed.
```
Comment (ilappe) [inline Packs/SplunkPy/ReleaseNotes/3_1_39.md]: please put more informative RN
it was related to drilldown etc.

File: Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py [hunk 2]
```diff
@@ -78,7 +78,7 @@
 QUERY_SEARCH = 'query_search'
 INCIDENT_CREATED = 'incident_created'
 
-DRILLDOWN_REGEX = r'([^\s\$]+)=(\$[^\$]+\$)|(\$[^\$]+\$)'
+DRILLDOWN_REGEX = r'([^\s\$]+)\s*=\s*["]*(\$[^\s\$]+\$)["]*|"*(\$[^\s\$]+\$)"*'
```
Comment (ilappe) [inline Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py:81]: as discussed
the `"` should be with the `?` 
and add the `\\`
something like that 
`([^\s\$]+)\s*=\s*"?(\$[^\s\$\\]+\$)"?|"?(\$[^\s\$\\]+\$)"?`

**Review by ilappe — CHANGES_REQUESTED** [review]
great work!!

**Review by dantavori — APPROVED** [review]

**Review by ilappe — APPROVED** [review]
great!

**Pull request merged**